toolbox/certbot/README.md

```diff
@@ -0,0 +1,52 @@
+# Certbot Certificate Renewal Script
+
+This folder contains a script to automate the renewal of Let's Encrypt SSL/TLS certificates for specific subdomains managed by the DTS Services HAProxy infrastructure. The script is designed to work with AWS Route53 DNS validation and retrieves credentials securely from 1Password.
+
+## Currently Supported Host Names
+
+This script is currently used for the following subdomains, but is not limited to these domains. Any DNS entry hosted in the DTS Route 53 zones may have certificates generated or renewed using this script.
```

Comment:
Just to check that I'm understanding this correctly 
>Any DNS entry hosted in the DTS Route 53 zones may have certificates generated or renewed using this script.

this means that this script could potentially be used to generate/renew certs for any of our DTS Route 53 zones? Or does this sentence imply that other entries are using this script, but we don't have a list of which ones?

Reply:
yep - it's totally hostname agnostic. you can use it for any hostname we have in our Route53 zones. I used it to do citybase URLs today. worked great.